application/frontend/src/routes/about.tsx

```diff
@@ -12,126 +12,69 @@ import {
   Image,
   Link,
   Text,
-} from '@chakra-ui/react'
-import { createFileRoute } from '@tanstack/react-router'
-import { FaUsers, FaGamepad } from 'react-icons/fa'
+} from '@chakra-ui/react';
+import { createFileRoute } from '@tanstack/react-router';
+import { FaUsers, FaGamepad } from 'react-icons/fa';
 import { useFloatAnimation } from '../hooks/useFloatAnimation';
+import { useTeamAccordion, TeamMember } from '../hooks/useTeamAccordion';
+import { usePageMetadata } from '../hooks/usePageMetadata';
 export const Route = createFileRoute('/about')({
   component: AboutPage,
-})
+});
 
-const teamMembers = [
-  // Include your team members here
-  {
-    name: "Katy Lam",
-    role: "Team Lead",
-    bio: "",
-    hiddenLink: "",
-    image: "assets/images/team-members/katy.png",
+const TeamAccordion = ({ teamMembers }: { teamMembers: TeamMember[] }) => (
+  <Accordion allowToggle>
+    {teamMembers.map((member, index) => (
+      <AccordionItem key={index} border="none">
+        <h2>
+          <AccordionButton
+            _expanded={{ bg: 'ui.main', color: 'white' }}
+            _hover={{ bg: 'ui.main', color: 'white' }}
+          >
```

Comment:
I doesn't look like #51 was addressed?

Reply:
oh thats super fucking weird I wonder if that just got pulled from the merge or something, because you can see below it I am calling the hook and have it in the import line. Ill pull this take that out and push again

Reply:
an I know I pulled that out. Weird.

Reply:
From a158f39? Doesn't seem like it.